src/main/java/org/takes/facets/fork/TkFork.java

```diff
@@ -63,11 +63,17 @@
  * @see org.takes.facets.fork.FkRegex
  * @see org.takes.facets.fork.FkParams
  */
-@ToString(of = "forks")
-@EqualsAndHashCode(of = "forks")
+@ToString(of = TkFork.LOMBOK)
+@EqualsAndHashCode(of = TkFork.LOMBOK + "")
```

Comment:
@yegor256 I am ashamed to say, i have no idea. :)
The build is failing with the compiler warning "static-access" without that.